packages/graphql/src/contracts.js

```diff
@@ -336,71 +356,46 @@ export function setNetwork(net, customConfig) {
       decimals: '18',
       supply: '1000000000'
     })
+
+    if (shouldUseMobileBridge()) {
+      context.ognExec = new context.web3Exec.eth.Contract(
+        OriginTokenContract.abi,
+        context.ogn._address
+      )
+    }
```

Comment:
Do we need `context.ognExec` without mobile bridge?

Reply:
Yeah this part is where I feel I could still do a better job with refactoring, but haven't tackled that part. This still gets set in the `setMetaMask` function: https://github.com/OriginProtocol/origin/blob/master/packages/graphql/src/contracts.js#L403-L431 
I didn't refactor it out into separate methods because `setMetaMask` gets called by exported `toggleMetaMask`. Meaning that besides working as part of the setup, it also needs to be toggle-able for on the fly enable/disable of Metamask.

Reply:
I am very open to suggestions how this could be more neatly solved.